pkg/graveler/graveler.go

```diff
@@ -289,7 +289,7 @@ func (c Commit) Identity() []byte {
 
 // CommitRecord holds CommitID with the associated Commit data
 type CommitRecord struct {
-	CommitID CommitID `db:"id"`
+	CommitID CommitID
```

Comment:
Something went wrong here with the linting/comments

Reply:
done